src/components/journeys/JourneyInstancesDataTable/index.tsx

```diff
@@ -9,7 +9,8 @@ import { ZetkinJourneyInstance } from 'types/zetkin';
 import { FunctionComponent, useState } from 'react';
 
 import { JourneyTagColumnData } from 'utils/journeyInstanceUtils';
-import UserConfigurableDataGrid from 'components/UserConfigurableDataGrid';
+import useConfigurableDataGridColumns from 'components/UserConfigurableDataGrid/useConfigurableDataGridColumns';
+import { useModelsFromQueryString } from 'components/UserConfigurableDataGrid/useModelsFromQueryString';
```

Comment:
Why is one default-exported, and the other a named export?

Reply:
I realize this is likely my design but I'm not sure why it's the case, and seeing it here makes me think they should both just be default exports from their respective module.

Reply:
yep haha its yours, but I'll change to default exports from both :)